backend/src/routes/module-federation.ts

```diff
@@ -97,9 +97,9 @@ export default async (fastify: KubeFastifyInstance): Promise<void> => {
       const serviceNamespace =
         getEnvVar('SERVICE_NAMESPACE') ?? service.namespace ?? process.env.OC_PROJECT;
       const servicePort = getEnvVar('SERVICE_PORT') ?? service.port;
-      const host = getEnvVar('LOCAL_HOST') ?? local.host ?? 'localhost';
```

Comment:
Great catch, this was breaking the dashboard in production if local was missing.